packages/reason/src/bodyRenderer.re

```diff
@@ -6,10 +6,17 @@ let make body::(body: PhenomicContent.jsBody) _children => {
     | PhenomicContent.String string => ReasonReact.stringToElement string
     | PhenomicContent.Element tag originalProps reasonChildren =>
       let props = ReactDOMRe.objToDOMProps originalProps;
-      ReactDOMRe.createElement
-        tag
-        ::props
-        [|ReasonReact.arrayToElement (Array.of_list (List.map renderChild reasonChildren))|]
+      switch tag {
+      | "a" =>
+        <Link toURL=[%bs.raw {| child[1].href |}]>
+          ([%bs.raw {|child[2]|}] |> Array.of_list |> ReasonReact.arrayToElement)
+        </Link>
```

Comment:
could we change this to use the thing I made in react BodyRenderer? The component I created handle properly external & relative url. It would be nice to be able to reuse it somehow as relying on rr Link is really not reliable in the real work (where markdown can give us any sort of link)